application/backend/src/main/java/cmpe451/group12/beabee/goalspace/controller/GroupGoalController.java

```diff
@@ -118,7 +118,7 @@ public MessageResponse leaveGroupGoal(
 
     /************************************** SUBGOALS *******/
     @ApiOperation(value = "Create a subgoal under a group goal.")
-    @PostMapping("/subgoal")
+    @GetMapping("/subgoal")
```

Comment:
Are you sure about this?

This is a create endpoint, so it should be a @PostMapping

Reply:
Yep mixed the create and get mapping types fixed it.